go.mod

```diff
@@ -3,26 +3,27 @@ module github.com/giantswarm/cert-operator
 go 1.13
 
 require (
-	github.com/giantswarm/apiextensions v0.4.1
-	github.com/giantswarm/certs v0.2.0
-	github.com/giantswarm/exporterkit v0.2.0
-	github.com/giantswarm/k8sclient v0.2.0
+	github.com/giantswarm/apiextensions/v3 v3.17.0
+	github.com/giantswarm/certs/v3 v3.1.1
+	github.com/giantswarm/exporterkit v0.2.1
+	github.com/giantswarm/k8sclient/v5 v5.0.0
 	github.com/giantswarm/microendpoint v0.2.0
 	github.com/giantswarm/microerror v0.3.0
 	github.com/giantswarm/microkit v0.2.2
 	github.com/giantswarm/micrologger v0.5.0
-	github.com/giantswarm/operatorkit v0.2.1
+	github.com/giantswarm/operatorkit/v4 v4.0.0
```

Comment:
I recommend using v4.2.0 as there have been a few important fixes in the last two minors.

Reply:
Good idea, included that as well